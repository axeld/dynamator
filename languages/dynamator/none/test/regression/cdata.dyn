<dynamator language="none">
  <tag tag="is">
    <attr name="added">
      <content>true</content>
    </attr>
  </tag>
</dynamator>
