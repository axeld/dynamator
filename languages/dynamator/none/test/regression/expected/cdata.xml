<?xml version="1.0" encoding="ISO-8859-1" standalone="yes"?><body>
<![CDATA[
This <is/> cdata.
]]>
  <is added="true"></is>
</body>
<!-- generated by Dynamator Fri Jan 30 21:32:55 CST 2004-->
